src/index.tsx

```diff
@@ -408,6 +449,8 @@ class App extends React.Component<{}, AppState> {
       event.key === "ArrowUp" ||
       event.key === "ArrowDown"
     ) {
+      addOnBeforeUnload();
```

Comment:
If you know that something has changed, why not run the save instead of telling the user to do so?